src/entities/Deployment.ts

```diff
@@ -15,9 +15,23 @@ interface TriggeredPlanSummary {
 }
 
 export interface DeploymentProperties {
-  committer: string | undefined;
-  repositoryAction: string | undefined;
-  repositoryBranchName: string | undefined;
-  repositoryRevisionNumber: string | undefined;
```

Comment:
Was the revision number the commit hash or something? And does it match to one of the properties below?

Reply:
its mapped to the top level 'revision' instead of being in the properties map.  There is a value we support in the properties map as well but I'm nor sure we need to double send it.